docs/src/main/asciidoc/maven-tooling.adoc

```diff
@@ -952,3 +952,144 @@ the `${maven.multiModuleProjectDirectory}` will be pointing to the directory fro
+[WARNING] quarkus:update goal is experimental, its options and output might change in future versions
+[INFO] 
+[INFO] Recommended Quarkus platform BOM updates: <1>
+[INFO] Update: io.quarkus.platform:quarkus-bom:pom:2.7.1.Final -> {quarkus-version}
+[INFO] Update: io.quarkus.platform:quarkus-kogito-bom:pom:2.7.1.Final -> {quarkus-version}
+[INFO] Update: io.quarkus.platform:quarkus-camel-bom:pom:2.7.1.Final -> {quarkus-version}
+----
+
+<1> A list of currently recommended Quarkus platform BOM updates
+
+NOTE: Typically, a single project property will be used to manage all the Quarkus platform BOMs but the implementation isn't currently smart enough to point that out and will report updates for each BOM individually.
+
+If we modify the project to remove all the Kogito extensions from the project, change the version of the `quarkus-resteasy-reactive` extension to `2.6.3.Final` and downgrade `quarkus-prettytime` which is not included in the Quarkus platform BOMs to `0.2.0`, `quarkus:update` will report something like:
+
+[source,text,subs=attributes+]
+----
+[INFO] Recommended Quarkus platform BOM updates: <1>
+[INFO] Update: io.quarkus.platform:quarkus-bom:pom:2.7.1.Final -> {quarkus-version}
+[INFO] Update: io.quarkus.platform:quarkus-camel-bom:pom:2.7.1.Final -> {quarkus-version}
+[INFO] Remove: io.quarkus.platform:quarkus-kogito-bom:pom:2.7.1.Final <2>
+[INFO] 
+[INFO] Extensions from io.quarkus.platform:quarkus-bom:
+[INFO] Update: io.quarkus:quarkus-resteasy-reactive:2.6.3.Final -> remove version (managed) <3>
+[INFO] 
+[INFO] Extensions from registry.quarkus.io:
+[INFO] Update: io.quarkiverse.prettytime:quarkus-prettytime:0.2.0 -> 0.2.1 <4>
+----
+
+<1> A list of the currently recommended Quarkus platform BOM updates for the project
+<2> Given that the project does not include any Kogito extensions, the BOM import is recommended to be removed
+<3> An outdated version of the `quarkus-resteasy-reactive` is recommended to be removed in favor of the one managed by the `quarkus-bom`
+<4> The latest compatible version of the `quarkus-prettytime` extension
+
+=== Quarkus project rectify
+
+As was mentioned above, `quarkus:info`, besides reporting Quarkus platform and extension versions, performs a quick version alignment check, to make sure the extension versions used in the project are compatible with the imported Quarkus platform BOMs. If misalignments are detected, the following warning message will be logged:
+
+[source,text,subs=attributes+]
+----
+[WARNING] Non-recommended Quarkus platform BOM and/or extension versions were found. For more details, please, execute 'mvn quarkus:update -Drectify'
+----
+
+The `rectify` option is enabled, the `quarkus:update`, instead of suggesting the latest recommended Quarkus version updates, will log update instructions to simply align the extension dependency versions found in the project with the currently imported Quarkus platform BOMs.
```

Comment:
```suggestion
When the `rectify` option is enabled, `quarkus:update`, instead of suggesting the latest recommended Quarkus version updates, will log update instructions to simply align the extension dependency versions found in the project with the currently imported Quarkus platform BOMs.
```